### DISCUSSION TITLE
Hovers that appear in markdown preview don't disappear when switching back to editing

### DISCUSSION THREAD

**Discussion by Kinrany** [community/community#44688] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

This element that appears on hover cannot be closed and prevents the preview tab from being opened:

<img width="1044" alt="Screenshot 2023-01-18 at 20 14 51" src="https://user-images.githubusercontent.com/1634186/213250819-a2975767-b26f-4a73-8eb8-f28a17abc82f.png">

**Comment by SilasRodrigues19** [comment]
Hey @Kinrany 

I had this problem these days and I solved it by reloading the page and the cache, then I typed again.

**Comment by Raboo** [comment, marked as answer]
It seems that Github is aware of the problem and working on it.
https://github.com/orgs/community/discussions/44630
